src/sumo/wrapper/sumo_client.py

```diff
@@ -12,6 +12,8 @@
 
 logger = logging.getLogger("sumo.wrapper")
 
+DEFAULT_TIMEOUT = httpx.Timeout(10.0)
```

Comment:
We can try, but by default it was 5 seconds and that was too low for some aggregation operations

Reply:
Should we increase it more? 20s?

Reply:
Yes let's try 20